docs/java-helloworld.md

```diff
@@ -0,0 +1,145 @@
+---
+id: java-helloworld
+title: Java SDK "Hello World!" tutorial
+sidebar_label: Java "Hello World!"
+tags: helloworld, java, sdk, intellij, gradle, tutorial
+---
+
+This tutorial is aimed at developers who are new to [Temporal](/docs/overview), beginner Java programmers, or both. It will teach you how to build a simple "Hellow World!" Temporal Workflow application using the [Temporal Java SDK](https://github.com/temporalio/java-sdk) as well as a few basics, such as:
+
+- How to set up your development environment and its dependencies.
+- How to define an [Activity](/docs/activities) and a [Workflow](/docs/workflows).
+- How to create and execute a Worker program and a Workflow starter program.
+- Where to view Workflow details.
+
+## Prerequisites
+
+Make sure you have the tutorial [prerequisites](/docs/java-sdk-tutorial-prerequisites) setup.
+
+## Project setup
```

Comment:
Should we consider adding instructions for command line only experience? For example a new gradle project can be easily generated using command line commands.

Another option to consider is to use maven instead of gradle.